pages/index.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Friends of Homeless Animals</title>
+  <link href="http://fonts.googleapis.com/css?family=Roboto:400,300,500,700,300italic,400italic,500italic,700italic" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="/Users/jenniferule/Documents/final-project/css/reset.css">
+  <link rel="stylesheet" href="/Users/jenniferule/Documents/final-project/css/main-styling.css">
+</head>
+
+<header class="clearfix">
+  <div class="logo">
+    <a href="pages/index.html"><img src="/Users/jenniferule/Documents/final-project/fohalogo.gif" /></a>
+  </div>
+  <nav>
+    <ul>
+      <li>
+        <a href="pages/index.html">Home</a>
+      </li>
+      <li>
+        <a href="pages/about.html">About</a>
+      </li>
+      <li>
+        <a href="pages/store.html">Resale Shop</a>
+      </li>
+      <li>
+        <a href="pages/contact.html">Contact</a>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<section class="banner">
+  <img src="http://placehold.it/950x200?text=Banner" />
+</section>
+
+<section class="buttons">
+  <div class="wrapper">
+    <div class="action1-4">
+      <a href="pages/adopt.html"><h1>Adopt</h1></a>
+      <p>Interested in adopting one of our dogs?</p>
+    </div>
+    <div class="action1-4">
+      <a href="pages/volunteer.html"><h1>Volunteer</h1></a>
+      <p>Want to spend time with our pups?</p>
+    </div>
+    <div class="action1-4">
+      <a href="pages/foster.html"><h1>Foster</h1></a>
+      <p>Looking to foster one our pups?</p>
+    </div>
+    <div class="action1-4">
+      <a href="pages/donate.html"><h1>Donate</h1></a>
+      <p>Would you like to donate or sponsor a dog?</p>
+    </div>
+  </div>
+</section>
+
+<div class="wrapper">
+  <div class="welcomemsg">
```

Comment:
snake case your classes with .welcome-msg instead of welcomemsg. The first is easier to read.